assets/js/blocks/stock-filter/style.scss

```
@@ -39,3 +35,10 @@
 		}
 	}
 }
+
+.editor-styles-wrapper .wc-block-stock-filter .wp-block-button__link,
+.wc-block-stock-filter .wc-block-stock-filter__button {
```

Is it expected that the editor selector is different than the frontend one? I would expect them to be the same:

```suggestion
.editor-styles-wrapper .wc-block-stock-filter .wc-block-stock-filter__button,
.wc-block-stock-filter .wc-block-stock-filter__button {
```